en/mkdocs.yml

```diff
@@ -261,13 +261,13 @@ nav:
          # - Kafka Consumer and Producer: learn/samples/
           - Message Filtering: learn/samples/message-filtering.md
           - Periodical Scheduled Tasks: learn/samples/periodical-scheduled-tasks.md
-         # - Protocol Switching: learn/samples/
-         # - Proxying a REST Service: learn/samples/
-         # - Proxying a SOAP Service: learn/samples/
+          - Protocol Switching: learn/samples/protocol-switching.md
+          - Proxying a REST Service: learn/samples/proxying-REST-service.md
+          - Proxying a SOAP Service: learn/samples/proxying-SOAP-service.md
```

Comment:
Shall we use all lowercase for consistency in our naming conventions for file and image names?

Reply:
done